src/download_pdfs.py

```diff
@@ -0,0 +1,47 @@
+import os
+
+import yaml
+from playwright.sync_api import Playwright, sync_playwright
+
+
+def run(playwright: Playwright) -> None:
+    with open("weekly_report.yaml", "r") as yaml_file:
+        teams_data = yaml.safe_load(yaml_file)
+
+    download_path = teams_data["config"]["Download_path"]
+
+    browser = playwright.chromium.launch(headless=False)
```

Comment:
Shouldn't `headless` be set to `True`?  (At least, when we're in production, I don't think we actually need to see what the tool is doing...right?)